### PR TITLE
[2.3] [Contributing] [CS] Added missing docblocks in code snippet

### PR DESCRIPTION
**Pull request by phansys**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to |  |
| Fixed tickets |  |

### CODE REVIEW

File: contributing/code/standards.rst [hunk 1]
```diff
@@ -48,7 +51,9 @@ example containing most features described below:
         }
 
         /**
-         * @param string $dummy Some argument description
+         * Transforms the input given as first argument.
+         *
+         * @param string $dummy   Some argument description
          * @param array  $options
```
Comment (xabbuh) [inline contributing/code/standards.rst]: What about a description for the `$options` argument and the exception?

Comment (phansys) [inline contributing/code/standards.rst]: @xabbuh, updated.

**Comment by WouterJ** [comment]
:+1: (although we have a very weird example here now, we might need to rewrite it and have something more logical in the future...)

**Comment by SpacePossum** [comment]
:+1: 
(kindly note that this PR targets 2.7 while the title states 2.3)

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you Javier.

**Comment by phansys** [comment]
Thank you @xabbuh!

**Pull request closed without merging**